### ISSUE TITLE
Input files

### ISSUE THREAD

**Issue by holly-t** [kassonlab/sample_restraint#44]
The input files for running the example restrained-ensemble.py don't seem be provided. Could these be added? Thanks.

**Comment by eirrgang** [comment]
True, they were left as an exercise to the reader at the time. The actual files used in the paper may be available as supplemental data now. If not, we should be able to provide reasonable sample input (or a script that can generate it).

**Comment by eirrgang** [comment]
Update: we are working on preparing a repository of a bunch of data from the research that script was used in. Later this fall I expect to have better tutorial materials available, including more manageable toy systems to experiment with. In the mean time, if you would like a Google Drive link to the original TPR input file, please send us an email: info@gmxapi.org
